polyfill.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+module.exports = function getPolyfill() {
+	var implemented = 'from' in Array && (function () {
```

Comment:
instead of all these `in` checks, would it be better to test that it's callable?